builder/dockerfile/dispatchers.go

```diff
@@ -279,12 +280,31 @@ func workdir(b *Builder, args []string, attributes map[string]bool, original str
 		return err
 	}
 
-	// NOTE: You won't find the "mkdir" for the directory in here. Rather we
+	comment := fmt.Sprintf("WORKDIR %v", b.runConfig.WorkingDir)
```

Comment:
I know this was this way before, but should probably just be `comment := "WORKDIR " + b.runConfig.WorkingDir`